src/cloudservicesuploadadapter.js

```diff
@@ -26,7 +27,7 @@ export default class CloudServicesUploadAdapter extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ FileRepository ];
+		return [ FileRepository, CloudServicesUploadAdapter._CloudServices ];
```

Comment:
We don't use to mock plugins. I would mock `CloudServices` class in `CloudServices` plugin instead.

Reply:
Fixed.